<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetaryImagerMainWindow</class>
 <widget class="QMainWindow" name="PlanetaryImagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planetary Imager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="image">
      <property name="interactive">
       <bool>false</bool>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
      <property name="optimizationFlags">
       <set>QGraphicsView::DontAdjustForAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <widget class="QMenu" name="menu_device_load">
     <property name="title">
      <string>&amp;Load...</string>
     </property>
    </widget>
    <addaction name="action_devices_rescan"/>
    <addaction name="menu_device_load"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuPanels">
    <property name="title">
     <string>Pa&amp;nels</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChip_Info"/>
    <addaction name="actionCamera_Settings"/>
    <addaction name="actionRecording"/>
    <addaction name="actionHide_all"/>
    <addaction name="actionShow_all"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionActual_Size"/>
    <addaction name="actionFit_to_window"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuPanels"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="chipInfoWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Camera Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="camera_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chip Size (mm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>133</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="camera_chip_size">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pixels Size (um)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>133</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="camera_pixels_size">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>133</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="camera_resolution">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>bpp</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="camera_bpp">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>108</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="camera_settings">
   <property name="windowTitle">
    <string>Settin&amp;gs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QWidget" name="settings_container" native="true"/>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="recording">
   <property name="windowTitle">
    <string>&amp;Recording</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>File name prefix</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QPushButton" name="start_recording">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="stop_recording">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QLineEdit" name="filePrefix"/>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="save_file_buffered">
       <property name="text">
        <string>Buffered disk write</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLineEdit" name="saveDirectory"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Frames Limit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="saveFramesLimit">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>Infinite</string>
       </property>
       <item>
        <property name="text">
         <string>Infinite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>750</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5000</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_devices_rescan">
   <property name="text">
    <string>&amp;Rescan</string>
   </property>
  </action>
  <action name="action_device_use">
   <property name="text">
    <string>&amp;Load...</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionChip_Info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Chip Info</string>
   </property>
  </action>
  <action name="actionCamera_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera &amp;Settings</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Recording</string>
   </property>
  </action>
  <action name="actionStart_Live">
   <property name="text">
    <string>Start Live</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionActual_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Actual Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Fit to window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionPickFolder">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/folder.png</normaloff>:/resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionHide_all">
   <property name="text">
    <string>&amp;Hide all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionShow_all">
   <property name="text">
    <string>Sh&amp;ow all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
